pkg/apis/devops/v1alpha1/s2ibuilder_types.go

```diff
@@ -422,6 +422,12 @@ type S2iConfig struct {
 
 	// Whether output build result to status.
 	OutputBuildResult bool `json:"outputBuildResult,omitempty"`
+
```

Comment:
I recommend adding a token field here.
Avoiding unauthorized people can trigger builds

Reply:
add secret code to authenticate